### DISCUSSION TITLE
Using psycopg3 in PostgresHook

### DISCUSSION THREAD

**Discussion by mfatemipour** [apache/airflow#44453] [General]
### Description

Hi, I want to ask is there any plan to use psycopg3 instead of psycopg2 in PostgresHook? 

### Use case/motivation

There are a lot of performance improvements in psycopg3

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
No plans as far as I am concerned but we have no "plans" for that. In Open Source things are done when someone does it. Ant you are absolutely free to propose a PR. Airflow has > 3000 contributors and you can become one of them easily.

BTW. When you ask a question, you should not crete an issue -> as we explain in the template which you got when you attempt to create an issue, you should create a discussion instead. I convert it to a discussion now in case more discussion is neded.

**Reply by mfatemipour** [reply]
Hi,

Thank you for your reply. I couldn't find any related information in previous issues and discussions. To avoid duplicating efforts and to understand if there are any blockers from the **package authors**, I have asked this question.
